google/cloud/storage/list_hmac_keys_reader.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_STORAGE_LIST_HMAC_KEYS_READER_H_
+#define GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_STORAGE_LIST_HMAC_KEYS_READER_H_
+
+#include "google/cloud/status_or.h"
+#include "google/cloud/storage/internal/hmac_key_requests.h"
+#include "google/cloud/storage/internal/range_from_pagination.h"
+#include "google/cloud/storage/internal/raw_client.h"
+#include <iterator>
+
+namespace google {
+namespace cloud {
+namespace storage {
+inline namespace STORAGE_CLIENT_NS {
```

Comment:
#include "google/cloud/storage/version.h"

Reply:
Done.  But we have 3 other headers in this file that already *must* include `storage/version.h`.

Reply:
Granted, although it is nice if everything is independent.